packages/data-layer/src/queries.ts

```diff
@@ -70,6 +72,7 @@ export const getProgramById = gql`
 export const getProjectById = gql`
   query ($alloVersion: [String!]!, $projectId: String!) {
     projects(
+      first: 1
```

Comment:
since we have canonical and linked projects, we need to get all projects of one project id to get all the information we need.

currently we have logic in data-layer to merge the canonical and linked projects. we could move this logic to the indexer, but currently we should increase the limit.

Reply:
fixed, this query should be called `getProjectsById`.